controllers/gslb_controller.go

```diff
@@ -85,6 +86,7 @@ func (r *GslbReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	if err != nil {
 		return result.RequeueError(fmt.Errorf("resolving spec (%s)", err))
 	}
+	log.Debug().Msgf("resolved strategy:\n%s", str.ToString(gslb.Spec.Strategy))
```

Comment:
I'd avoid using new line (`\n`). It might beautify output in console, but when it comes to logging infra, additional formatting might complicate log parsing and further analysis.